tRecorderApi/api/file_transfer/FileUtility.py

```diff
@@ -205,6 +205,34 @@ def open_manifest_file(directory):
         except Exception as e:
             shutil.rmtree(directory)
             logger.error("Error: ", str(e))
+            
+    @staticmethod
+    def open_localization_file():
+        base_dir = os.path.dirname(os.path.dirname(
+            os.path.dirname(os.path.abspath(__file__))))
```

Comment:
It looks like you're trying to get the third directory up from the file.   Another way to do this that might be easier to read would be `str(Path(s).parents[2])`

Reply:
Thanks @PurpleGuitar!  Good recommendation